weaviate/data/crud_data.py

```diff
@@ -488,6 +501,12 @@ def get(
         if uuid is not None:
             path += "/" + get_valid_uuid(uuid)
 
+        if consistency_level is not None:
+            params["consistency_level"] = consistency_level
```

Comment:
Since python does not check the type annotation at run time (unless you use a library to do this), this would result in setting the `consistency_level` to any value. 

1. Should we add a check? @dirkkul do you know of an easier to validate this? Should we allow lower case values or not? if not I guess we could do this:
```python
if consistency_level is not None:
   if consistency_level not in ConsistencyLevel:
      raise ValueError("<some message and acceptable values>")
   params["consistency_level"] = consistency_level
```
2. Or should we leave it up to Weaviate to respond with an error? In this caseI guess we do not need the `ConsistencyLevel` at all and we could just use `str`.

Any thoughts on this?

Reply:
Uhhh "in" was deprecated with Enums in python 3.8, why would they do that o.O

[here are alternatives](https://stackoverflow.com/questions/63335753/how-to-check-if-string-exists-in-enum-of-strings) if that is too annoying let's go back to strings.

One of the big upsides of enums is that there is typesafety, and a) your IDE can autocomplete the parameter and b) typecheckers like mypy can make sure that you've got the right type.

Reply:
Agreed, kept the enum and added support for `in` statements